docs/sources/installation/configuration.md

```diff
@@ -29,18 +29,15 @@ A common problem is forgetting to uncomment a line in the `custom.ini` (or `graf
 - Custom configuration from `$WORKING_DIR/conf/custom.ini`
 - The custom configuration file path can be overridden using the `--config` parameter
 
-> **Note.** If you have installed Grafana using the `deb` or `rpm`
-> packages, then your configuration file is located at
-> `/etc/grafana/grafana.ini` and a separate `custom.ini` is not
-> used. This path is specified in the Grafana
-> init.d script using `--config` file parameter.
+**Note:** If you have installed Grafana using the `deb` or `rpm` packages, then your configuration file is located at `/etc/grafana/grafana.ini` and a separate `custom.ini` is not used. This path is specified in the Grafana init.d script using `--config` file parameter.
+
+**Debian and Ubuntu:** When you upgrade Grafana, the default `grafana.ini` is overwritten. We recommend that you save your custom config changes in a file named `<grafana_install_dir>/conf/custom.ini`. This allows you to upgrade Grafana without risking losing your configuration changes.
 
```

Comment:
The note in line 34 was from the Updating page. It seems like it might conflict with the previous note that was already in the Configuration page. Please review.